packages/navigation/src/hooks/useWebviewNavigate.ts

```diff
@@ -11,12 +11,12 @@ import {
   useNavigation,
   useRoute,
   CommonActions,
+  LinkingContext,
 } from '@react-navigation/native';
-import LinkingContext from '@react-navigation/native/src/LinkingContext';
```

Comment:
If we still want to support v6 (which I think we should), do we need to do a custom export here as well similar to `extractPathFromURL`?

Reply:
I checked that before and `LinkingContext` export changed in v7, but in both v6 and v7 it can be simply imported from `'@react-navigation/native'`, so there's no need for a custom export